src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -0,0 +1,359 @@
+                "The passenger details logbook is empty.", " ");
+    }
+
+    //@@author Franky4566
+    public void showEmptyFlightList() {
+        System.out.printf("| %80s %-45s |\n"
+                        + "+--------------------------------------------------------------------"
+                        + "--------------------------------------------------------------+\n",
+                "The Flight schedule is empty.", " ");
+    }
+
+    public String getFlightNotFoundMessage(String flightNum) {
+        return "FLIGHT " + flightNum + " NOT FOUND.";
+    }
+
+    public String showEmptyDescriptionMessage() {
+        return "oops! The description is empty :(";
+    }
+
+    public void showFlightAddedMessage() {
+        System.out.println("Flight added!");
+    }
+
+    //@@author JordanKwua
+    //@@author JordanKwua
+    public void showFlightRemovedMessage(String flightNum) {
+        System.out.println("FLIGHT " + flightNum + " HAS BEEN DELETED.");
+    }
+
+    public void showListOfFlights(ArrayList<FlightInfo> flightList) {
+        showFlightListHeader();
+        checkEmptyFlightList(flightList);
+        for (FlightInfo flight : flightList) {
+            System.out.println("\n+-----------------------------------------------------------"
+                    + "-----------------------------------------------------------------------+"
+                    + flight);
+        }
+    }
+
+    public void checkEmptyFlightList(ArrayList<FlightInfo> flightList) {
+        int numOfFlights = flightList.size();
+        if (numOfFlights == EMPTY_FLIGHT_LIST) {
+            showEmptyFlightList();
+        }
+    }
+
+    //@@author ivanthengwr
+    public String getLoggerError() {
+        return "File logger failed to work.";
+    }
+
+    public String getLoggerStartUpMessage() {
+        return "Going to start processing";
+    }
+
+    //@@author JordanKwua
+    public String getWrongFlightFormatErrorMessage() {
+        return "Flight number should start with 2 letters and trail with 1-4 numbers.";
+    }
+
+    //@@author shengiv
+    public String getMissingDetailsError() {
+        return "Please fill up details for all fields.\n Invalid input. Try again!";
+    }
+
+    public void showUpdatedFlightNumber(String flightNum, String newFlightNum) {
+        System.out.println("Flight number of flight " + flightNum + " is updated to " + newFlightNum + ".");
+    }
+
+    public void showUpdatedGateNumber(String flightNum, String newGateNum) {
+        System.out.println("Gate number of flight " + flightNum + " is updated to " + newGateNum + ".");
+    }
+
+    //@@author Franky4566
+    public void showUpdatedDepartureTime(String flightNum, String oldDepartureTime, String newDepartureTime) {
+        System.out.println("Departure time of flight " + flightNum + " is delayed from "
+                + oldDepartureTime + " to " + newDepartureTime + ".");
```

Comment:
Avoid magic string. Consider putting the string into a constant. One way to circumvent having mutiple similiar strings (or concatinated string) is to use string formatting. For example, the constant string could be `FLIGHT_DEPART_MSG = "Departure time of flight %d is delayed from %t to %t."`

When using the variable: 
```
string outputMsg = String.format (FLIGHT_DEPARTURE_MSG, flightNum, oldDepartureTime, newDepartureTime);
System.out.println(outputMsg);
```